draft-ietf-masque-h3-datagram.md

```diff
@@ -194,36 +203,49 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
+## HTTP Datagrams using Capsules
+
+When HTTP/3 Datagrams are unavailable or undesirable, HTTP Datagrams can be sent
+using the Capsule Protocol, see {{datagram-capsule}}.
+
+
 # Capsules {#capsule}
 
 This specification introduces the Capsule Protocol. The Capsule Protocol is a
 sequence of type-length-value tuples that new HTTP Upgrade Tokens (see {{Section
```

Comment:
See below re: focus on upgrade tokens. It's extensions that are choosing to use this mechanism, not the tokens themselves. They're just identifiers.

Reply:
Fixed